vulns/GHSA-9qr9-h5gf-34mp.json

```diff
@@ -0,0 +1,177 @@
+{
+  "schema_version": "1.6.7",
+  "id": "GHSA-9qr9-h5gf-34mp",
+  "modified": "2025-12-04T20:07:06Z",
+  "published": "2025-12-03T19:07:11Z",
+  "related": [
```

Comment:
You might want to use `alias` instead so that the linkage to CISA KEV shows up (as called out in this comment) https://github.com/aquasecurity/trivy-db/issues/597#issuecomment-3630489231